jackson/src/com/treode/jackson/notification.scala

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2014 Treode, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+
+package com.treode.jackson
+
+import com.fasterxml.jackson.core.JsonGenerator
+import com.fasterxml.jackson.databind.SerializerProvider
+import com.fasterxml.jackson.databind.ser.std.StdSerializer
+import com.treode.notify.Notification
+
+object NotificationSerializer extends StdSerializer [Notification [_]] (classOf [Notification [_]]) {
+  def serialize (value: Notification [_], jgen: JsonGenerator, provider: SerializerProvider) {
+    jgen.writeString (value.toString)
```

Comment:
This should be more like

```
    jgen.writeStartArray()
    for (message <- value.messages)
      codec.writeValue (jgen, message)
    jgen.writeEndArray()
```

To make that work, you also need to define a serializer for Message.